create/project_config.go

```diff
@@ -0,0 +1,61 @@
+package create
+
+import (
+	"context"
+	"time"
+
+	apps "github.com/ninech/apis/apps/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/api/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+type configCmd struct {
+	Wait        bool              `default:"true" help:"Wait until API Project Config is fully created."`
+	WaitTimeout time.Duration     `default:"10s" help:"Duration to wait for API Project Config getting ready. Only relevant if wait is set."`
+	Size        string            `default:"micro" help:"Size of the app."`
+	Port        int32             `default:"8080" help:"Port the app is listening on."`
+	Replicas    int32             `default:"1" help:"Amount of replicas of the running app."`
```

Comment:
Not sure about these defaults, seems very dangerous. If I do `nctl create config --replicas=3`, all my apps will suddenly listen on port `8080` (unless the app overrides it) even though I never intended to configure that. IMO just the args defined by the user should be set in the config and the rest should be `nil`.

Reply:
I completely agree

Reply:
The defaults are still there, I guess we need to remove them else kong will just set the flags.

Reply:
my bad - sorry.